Source/WebCore/PAL/pal/PALSwift/UnsafeOverlays.swift

```diff
@@ -250,8 +251,11 @@ extension Curve25519.Signing.PublicKey {
         if signature.empty() || data.empty() {
             return false
         }
-        return self.isValidSignature(Data.temporaryDataFromSpan(spanNoCopy: signature), for: Data.temporaryDataFromSpan(spanNoCopy: data))
+        return self.isValidSignature(
+            Data.temporaryDataFromSpan(spanNoCopy: signature),
+            for: Data.temporaryDataFromSpan(spanNoCopy: data))
```

Comment:
I'd agree that the code looks better this way but we often refrain from making cosmetic changes to actual code since the revision history gets obscured by it. With this change, it'll look at a glance like you wrote this code in your patch but it is older.

Reply:
https://github.com/WebKit/WebKit/pull/27474#discussion_r1576409180